### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by jreidinger**
you can ignore auto corrections commit, as it is done by rubocop itself without my intervention.

I would like to pin up real bug fix found by rubocop which report unused variable with typo see:

### CODE REVIEW

File: src/lib/network/install_inf_convertor.rb [hunk 1]
```diff
@@ -318,31 +314,31 @@ def create_wlan_ifcfg
...
+      when "wep_open", "wep_restricted"
+        if wlan_key_type == "password"
+          type = "h:"
+        elsif wlan_key_type == "ascii"
+          type = "s:"
+        else
+          type = wlan_key_type[0] + ":"
+        end
+
+        wlan_auth_mode = wlan_auth == "wep-open" ? "open" : "sharedkey"
```
Comment (jreidinger) [inline src/lib/network/install_inf_convertor.rb]: this is actually bug fix, as original auth_mode is not used and below wlan_auth_mode crash.

**Comment by mchf** [comment]
one test didn't survive changes (lan_udev_auto)

**Comment by jreidinger** [comment]
I will fix test, I try to finish it before left to NUE, but it failed, so I will fix it tomorrow

**Comment by mchf** [comment]
I've learned something, tests pass and I have no more objections so LGTM

**Comment by jreidinger** [comment]
great

**Pull request merged**